hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/AbstractAmqpServiceIntegrationTest.java

```diff
@@ -102,6 +110,17 @@ public void initListener() {
         }
     }
 
+    protected void waitUntilEventMessagesAreDispatchedToTarget(final EventTopic... eventTopics) {
+        createConditionFactory().untilAsserted(() -> {
```

Comment:
May I ask you to set visibility of `waitUntilIsPresent(final Callable<Optional<T>> callable)` to `private`. I think it is not used outside this class